locale/zh/messages.json

```diff
@@ -1,1171 +1,1171 @@
-  "user will be deleted": "用户将被删除",
-  "will be deleted": "将被删除",
-  "your service's requirements ({requestedValue}).": "您服务的要求 ({requestedValue})。",
-  "your service's role ({requestedValue}).": "您服务的角色 ({requestedValue})。",
-  "{0, plural, one {# instance running out of {instancesCount}} other {# instances running out of {instancesCount}}}": "",
-  "{0}": "{0}",
-  "{0} (Change)": "{0}（更改）",
-  "{0} ({1} available)": "{0}（{1} 可用）",
-  "{0} Info": "{0} 信息",
-  "{0} Version": "{0} 版本",
-  "{0} is being installed.": "{0} 正在安装。",
-  "{0} will be deleted.": "{0} 将被删除。",
-  "{0} your group by {updateDescription} from the DC/OS CLI.": "从  DC/OS CLI 通过{updateDescription} {0} 组",
-  "{0} {packagesWord} found": "已找到 {0} {packagesWord}",
-  "{agentsAvailable} of {agentsTotal}": "{agentsAvailable} 的 {agentsTotal}",
-  "{backendCount} Total Backends": "{backendCount} 总后端",
-  "{clientCount} Total Clients": "{clientCount} 总客户端",
-  "{completedSteps} out of {totalSteps} steps completed.": "",
-  "{containerName} Path": "{containerName} 路径",
-  "{errorMsg} Go to <0>insert permission string</0> to enter a permission manually.": "{errorMsg} 转至 <0>插入权限字符串</0> 以手动输入权限。",
-  "{issueCount} issues need addressing": "{issueCount} 问题需要解决",
-  "{itemType} is currently locked by one or more deployments. Press the button again to forcefully change and deploy the new configuration.": "{itemType} 当前被一个或多个部署锁定。再次按下按钮，以强制更改和部署新配置。",
-  "{logName} Log is Currently Empty": "{logName} 日志当前为空",
-  "{numBackends} Backends": "{numBackends} 后端",
-  "{numClients} Clients": "{numClients} 客户端",
-  "{runningInstances, plural, one {# instance running out of {instancesCount}} other {# instances running out of {instancesCount}}}": "{runningInstances, plural, one {# 实例将用完 {instancesCount}} other {# 实例将用完 {instancesCount}}}",
-  "{runningInstances} {0} running out of {instancesCount}": "{runningInstances} {0} 将用完 {instancesCount}",
-  "{userLabel} will be removed from the {groupLabel} group.": "{userLabel} 将从 {groupLabel} 组移除。",
-  "{userName} will be removed from {groupLabel}.": "{userName} 将从 {groupLabel} 移除。",
-  "{watching} out of {totalFound}": "{watching} 用完 {totalFound}"
+{
+  "(AT BEGINNING OF FILE)": "（运行文件时）",
+  "(Clear)": "（清除）",
+  "(No data)": "（无数据）",
+  "(Optional. Default: 20): Number of seconds after which a health check is considered a failure regardless of the response.": "（可选。默认值：20）：运行状况检查被视为失败（无论响应如何）后的秒数。",
+  "(Optional. Default: 3): Number of consecutive health check failures after which the unhealthy instance should be killed. HTTP & TCP health checks: If this value is 0, instances will not be killed if they fail the health check.": "（可选。默认值：3）：不健康实例应被杀死之后运行状况检查连续失败的次数。HTTP 和 TCP 运行状况检查：如果此值为 0，那么尽管实例执行运行状况检查失败，也不会被杀死。",
+  "(Optional. Default: 300): Health check failures are ignored within this number of seconds or until the instance becomes healthy for the first time.": "（可选。默认值：300）：运行状况检查失败在此秒数内或直到实例
```

Comment:
In the new translations, "Arg"/"Args" is all caps. Is this intentional?

Reply:
Think so… They did the same with some other things.